src/components/CollaboratorModal/CollaboratorModal.tsx

```diff
@@ -24,13 +22,16 @@ import * as CollaboratorHooks from "hooks/collaboratorHooks"
 
 import { useSuccessToast } from "utils"
 
+interface CollaboratorModalProps extends Omit<ModalProps, "children"> {
+  siteName: string
+}
+
```

Comment:
fyi interface changed a lil - behaves like a fully fledged modal w/o children now (should be minor change)

Reply:
Updated via a rebase in d63380d.